app/assets/javascripts/suggest.js

```diff
@@ -6,22 +6,27 @@
         var $this, callback, timer;
         $this = $(this);
         callback = function() {
+          var js_suggest_selector, locale;
+          js_suggest_selector = $this.data("js-suggest");
+
+          if (js_suggest_selector.startsWith(".")) {
+            locale = $this.closest(".translatable-fields").data("locale");
+            js_suggest_selector += "[data-locale=" + locale + "]";
+          }
+
           $.ajax({
             url: $this.data("js-url"),
             data: {
               search: $this.val()
             },
             type: "GET",
             dataType: "html",
+            beforeSend: function() {
+              $(js_suggest_selector).removeClass("success").addClass("loading");
```

Comment:
Hi @javierm, as the CSS classes `success` and `loading` are for testing purposes and are quite generic class names. What do you think about adding the following prefixes `js-success`, `suggest-success` or maybe `test-success`? I can imagine forks with custom code where CSS styles are defined for the `.success` or `.loading` classes, which could break the functionality or change the suggestions styles unintentionally.

Is the `loading` class necessary, or maybe it is added for javascript clarity? Seems fine to me, just curious.

Reply:
I've changed them to `suggest-succes` and `suggest-loading` :+1:.

> Is the loading class necessary, or maybe it was added for javascript clarity?

Just for clarity. I thought it was a bit strange to keep the suggest class while new suggestions were loading (would happen in a second request if we didn't remove the class en `beforeSend`), and then I thought we could differentiate the case :thinking:.

Reply:
Thanks for the clarification and for the commit message update 🎖️ !